src/generators/openapi/generators/openapi-utils/convertApiBuilderPrimitiveType.ts

```diff
@@ -11,7 +11,10 @@ function convertApiBuilderPrimitiveType(type: ApiBuilderPrimitiveType) {
     case Kind.DECIMAL: return { type: 'number', format: 'float' };
     case Kind.DOUBLE: return { type: 'number', format: 'double' };
     case Kind.INTEGER: return { type: 'integer' };
-    case Kind.JSON: return { AnyValue: {} };
+    case Kind.JSON: return {
+      properties: {},
+      type: 'object',
```

Comment:
`json` in apibuilder is not necessarily an object - it can accept any type (number, string, null, object, array)